### PR TITLE
addes hover to card and nav link

### PR DESCRIPTION
**Pull request by anna-chovnova**
- [DEMO LINK](https://anna-chovnova.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: 300ms ease;
```
Comment (Zhanna-Osypenko) [inline src/styles/_catalog.scss:18]: transition: transform 300ms linear color 300ms;

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -14,6 +14,13 @@
...
+
+    &:hover {
+      width: 200px * 1.2;
+      height: 300px * 1.2;
+    }
```
Comment (Zhanna-Osypenko) [inline src/styles/_catalog.scss:23]: &:hover {
      transform: scale(1.2);
      color: #34568b;
    }

File: src/styles/_header.scss [hunk 3]
```diff
@@ -37,5 +36,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms ease;
```
Comment (VETALEK) [inline src/styles/_header.scss:39]: you need to specify the transition property just like you did in `.catalog__card` 😉

**Review by VETALEK — CHANGES_REQUESTED** [review]
GJ 👍

File: src/styles/_header.scss [hunk 4]
```diff
@@ -36,7 +36,7 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
-    transition: 300ms ease;
+    transition: transform 300ms linear, color 300ms linear;
```
Comment (VETALEK) [inline src/styles/_header.scss:39]: you don't need the `transform` here 😉

**Review by VETALEK — CHANGES_REQUESTED** [review]
almost 💪

File: src/styles/_header.scss [hunk 5]
```diff
@@ -37,5 +36,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms linear, color 300ms linear;
```
Comment (VETALEK) [inline src/styles/_header.scss:39]:
```suggestion
    transition: color 300ms linear;
```

only `color` 😉

**Review by VETALEK — CHANGES_REQUESTED** [review]
Last charge 💪

**Review by VETALEK — APPROVED** [review]
like 👍

**Pull request closed without merging**